XETEX_MAC uses deprecated functions from freetype here that we don't
compile.  In fact they only use it to query the system fonts.  Luckily,
our fontconfig (the alternative) does include those.

--- texk/xdvipdfmx/configure.in
+++ texk/xdvipdfmx/configure.in
@@ -106,21 +106,14 @@
 
 dnl Check for additonal libraries.
 
-JFK_HAS_APP_SERVICES
-if test "x$have_app_services" = xno; then
-  CHO_HAS_LIBFONTCONFIG
-fi
+CHO_HAS_LIBFONTCONFIG
 
-# check that it's valid to define XETEX and possibly XETEX_MAC
-if test "x$have_app_services" = xyes; then
-  AC_DEFINE([XETEX], [], [Define if part of a teTeX/TeX live build.])
-  AC_DEFINE([XETEX_MAC], [], [Define if part of a teTeX/TeX live build for Mac OS X.])
-elif test "x$have_libfontconfig" = xyes; then
+if test "x$have_libfontconfig" = xyes; then
   AC_DEFINE([XETEX])
 else
-  AC_MSG_ERROR([fontconfig library or ApplicationServices framework not found.
-This version of xdvipdfmx requires either ApplicationServices (on Mac OS X)
-or the fontconfig library (on other platforms) to be available, otherwise it
+  AC_MSG_ERROR([fontconfig library not found.
+This version of xdvipdfmx requires 
+the fontconfig library to be available, otherwise it
 cannot support XeTeX output using standard system-installed fonts.
     ])
 fi
